collector.go

```diff
@@ -0,0 +1,59 @@
+package versionbundle
+
+import (
+	"net/url"
+
+	"github.com/giantswarm/microerror"
+	"github.com/go-resty/resty"
+)
+
+type CollectorConfig struct {
+	RestClient *resty.Client
+
+	Endpoints []url.URL
+}
+
+func DefaultCollectorConfig() CollectorConfig {
+	return CollectorConfig{
+		RestClient: nil,
+
+		Endpoints: nil,
+	}
+}
+
+type Collector struct {
+	restClient *resty.Client
+
+	bundles []Bundle
```

Comment:
I don't get the ordering here - should `bundles` be at the bottom? If `endpoints` is configuration, and `bundles` is internal.

Reply:
- dependencies
- internals
- settings

Reply:
Ah, I normally roll with internals and settings the other way around. all good.